<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDialog</class>
 <widget class="QDialog" name="RecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RecordDialog</string>
  </property>
  <widget class="VideoShow" name="wdg_videoItem" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>591</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="videoGroup">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>151</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="cb_cam">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_desk">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>桌面</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_picinpic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>画中画</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_video">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_audio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>音频输入</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx_cam">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>默认视频设备</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbx_audio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>默认音频设备</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_voice">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="slider_volumn">
   <property name="geometry">
    <rect>
     <x>709</x>
     <y>350</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="value">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_recordBegin">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_recordEnd">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_res">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_savepath">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_address">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_savePath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_broadcast">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>560</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_savePath">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>497</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_rtmpUrl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>557</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_recPath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_recTime">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>450</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoShow</class>
   <extends>QWidget</extends>
   <header location="global">videoshow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
